odl/discr/discretization.py

```diff
@@ -316,6 +316,26 @@ def __setitem__(self, indices, values):
         else:
             self.ntuple.__setitem__(indices, values)
 
+    @property
+    def restriction(self):
+        """The restriction operator associated with this vector.
+
+        See Also
+        --------
+        RawDiscretization.restriction : For full description
+        """
+        return self.space.restriction(self.ntuple)
+
+    @property
+    def extension(self):
+        """The extension operator associated with this vector.
+
+        See Also
+        --------
+        RawDiscretization.extension : For full description
+        """
+        return self.space.extension(self.ntuple)
```

Comment:
See above. While this makes sense, the docstring isn't quite correct. If you apply it to the `ntuple` I wouldn't make it a property since this could possibly be expensive.